Stratis.Bitcoin/Features/MemoryPool/MempoolManager.cs

```diff
@@ -245,5 +253,18 @@ public Task RemoveForBlock(Block block, int blockHeight)
                 this.Validator.PerformanceCounter.SetMempoolDynamicSize(this.memPool.DynamicMemoryUsage());
             });
         }
+
+        /// <inheritdoc />
+        public async Task<UnspentOutputs> GetUnspentTransactionAsync(uint256 trxid)
+        {
+            var txInfo = await InfoAsync(trxid);
```

Comment:
Please read our code styling methods (available in the documentation folder)
we use this. for instance methods